.github/workflows/deploy-application.yml

```diff
@@ -157,3 +175,54 @@ jobs:
         run: |
           aws deploy wait deployment-successful --deployment-id $deployment_id
           echo "Deployment successful"
+
+  create-good-job-deployment:
```

Comment:
How similar is this to `create-thrust-deployment`? Could it be pulled out in to a shared template and then called twice for each type of deployment?